feature: add csrf token to the tasklist docs (#3672)
camunda/camunda-docs: docs/apis-tools/tasklist-api/tasklist-api-authentication.md

### Authentication via cookie

:::note

> When authenticating via cookie, it is necessary to note that Cross-Site Request Forgery (CSRF) protection must be disabled to allow this method of authentication. In a Self-Managed Camunda cluster, this can be done by setting the configuration property `camunda.tasklist.csrfPreventionEnabled` to `false`.

Reviewer: ```suggestion
When authenticating via cookie, note that Cross-Site Request Forgery (CSRF) protection must be disabled to allow this method of authentication. In a Camunda Self-Managed cluster, set the configuration property `camunda.tasklist.csrfPreventionEnabled` to `false`.
```
